### DISCUSSION TITLE
Improve upload session area

### DISCUSSION THREAD

**Discussion by jhdebruin** [djong1/gp3s#216] [Website improvements]
What to do with not compliant tracks , no user feedback
easier navigation after upload 

<img width="1268" alt="Screenshot 2023-08-30 at 21 46 59" src="https://github.com/djong1/gp3s/assets/109060128/8522e5fd-fd91-4e7e-896c-a8fb89506da5">

**Comment by jottevanger** [comment]
I'm a totally new user on the site and very much agree with this. There's no feedback at all, and the UI gives little indication of what to expect when you click anything. I've had no luck with uploading tracks but there's no ability to delete/cancel them nor clear indication when they've actually failed so it's very confusing. Terminology around sessions and tracks is used inconsistently or vaguely too, making it harder to know how they relate to each other

**Comment by djong1** [comment]
Thanks for your feedback, can you help us to improve. It would be very appreciated if you can create a drawing or a rough screen edit and make visible wat flow would work for you. Than we can implement this and make it better. Keep on sunbmitting feedback!

Verzonden vanuit Outlook voor iOS<https://aka.ms/o0ukef>
________________________________
Van: jottevanger ***@***.***>
Verzonden: Saturday, September 30, 2023 4:44:19 PM
Aan: djong1/gp3s ***@***.***>
CC: Subscribed ***@***.***>
Onderwerp: Re: [djong1/gp3s] Improve upload session area (Discussion #216)


I'm a totally new user on the site and very much agree with this. There's no feedback at all, and the UI gives little indication of what to expect when you click anything. I've had no luck with uploading tracks but there's no ability to delete/cancel them nor clear indication when they've actually failed so it's very confusing. Terminology around sessions and tracks is used inconsistently or vaguely too, making it harder to know how they relate to each other

**Comment by djong1** [comment]
@jhdebruin can you create a new UI proposal for this?

**Comment by jhdebruin** [comment]
@djong1 Yes, will make some mock-ups this week for it,  @jottevanger would be nice to share them here and get some feedback on it...
